core/src/main/java/com/amplifyframework/auth/options/AuthSignUpOptions.java

```diff
@@ -80,47 +75,73 @@ public boolean equals(Object obj) {
             return false;
         } else {
             AuthSignUpOptions authSignUpOptions = (AuthSignUpOptions) obj;
-            return ObjectsCompat.equals(getValidationData(), authSignUpOptions.getValidationData()) &&
-                    ObjectsCompat.equals(getUserAttributes(), authSignUpOptions.getUserAttributes());
+            return ObjectsCompat.equals(getUserAttributes(), authSignUpOptions.getUserAttributes());
         }
     }
 
+    /**
+     * When overriding, be sure to include the parent properties in the output string.
```

Comment:
--> `be sure to include userAttributes in the ...`

Reply:
Alright